tools/buildsys/src/builder.rs

```diff
@@ -247,11 +247,13 @@ impl DockerBuild {
         manifest: &Manifest,
         image_features: HashSet<ImageFeature>,
     ) -> Result<Self> {
-        let package = if let Some(name_override) = manifest.info().package_name() {
-            name_override.clone()
-        } else {
-            args.cargo_package_name
-        };
+        let package = manifest.info().package_name();
+        let per_package_dir = format!(
+            "{}/{}",
+            args.packages_dir.display(),
+            manifest.info().manifest_name()
```

Comment:
Are we able to use `package` here?
```suggestion
        let per_package_dir = format!(
            "{}/{}",
            args.packages_dir.display(),
            package,
```